runtime/vm/jvminitcommon.c

```diff
@@ -24,34 +24,34 @@
 #include "ut_j9vm.h"
 
 /**
- *  Walks the J9VMDllLoadInfo table looking for the entry for a specific library name. 
- *  
- *  @param aPool the pool holding the DLL info
- *  @param dllName the name of the dll we are looking for
- *  @returns J9VMDllLoadInfo for the dll requested or NULL if not found
+ * Walks the J9VMDllLoadInfo table looking for the entry for a specific library name.
+ *
+ * @param aPool the pool holding the DLL info
+ * @param dllName the name of the dll we are looking for
+ * @returns J9VMDllLoadInfo for the dll requested or NULL if not found
  */
-J9VMDllLoadInfo *findDllLoadInfo(J9Pool* aPool, const char* dllName) {
-	J9VMDllLoadInfo *anElement;
+J9VMDllLoadInfo *
+findDllLoadInfo(J9Pool *aPool, const char *dllName)
+{
+	J9VMDllLoadInfo *anElement = NULL;
 	pool_state aState;
-	char key[SMALL_STRING_BUF_SIZE];			/* Plenty big enough. Needs to be at least DLLNAME_LEN + 4. */
-
-	strncpy(key, dllName, SMALL_STRING_BUF_SIZE);
 
-	anElement = (J9VMDllLoadInfo*)pool_startDo(aPool, &aState);
-	while (anElement && (strcmp(anElement->dllName, key)!=0)) {
-		anElement = (J9VMDllLoadInfo*)pool_nextDo(&aState);
+	anElement = (J9VMDllLoadInfo *)pool_startDo(aPool, &aState);
+	while ((NULL != anElement) && (0 != strcmp(anElement->dllName, dllName))) {
+		anElement = (J9VMDllLoadInfo *)pool_nextDo(&aState);
 	}
 
 	/* If library not found and name does not end in major/minor version, prepend j9 and append version */
-	if (anElement==NULL && strstr(dllName, J9_DLL_VERSION_STRING)==NULL) {
-		memset(key, 0, SMALL_STRING_BUF_SIZE);
+	if ((NULL == anElement) && (NULL == strstr(dllName, J9_DLL_VERSION_STRING))) {
+		char key[SMALL_STRING_BUF_SIZE]; /* Plenty big enough. Needs to be at least DLLNAME_LEN + 4. */
+
 		strcpy(key, "j9");
 		safeCat(key, dllName, SMALL_STRING_BUF_SIZE);
```

Comment:
This change is more about the _removal_ of the use of `strncpy()` preceding the first loop in this function.

Reply:
The change itself is fine. The `use safeCat() instead` part of the commit comment implies the change modifies the code to use safeCat() instead of strncpy() but this isn't the case, it removes the unnecessary strncpy().

Reply:
Updated commit message.